api.planx.uk/modules/flows/findReplace/findReplace.test.ts

```diff
@@ -264,280 +264,4 @@ describe("string replacement", () => {
         expect(res.body.error).toMatch(/Failed to find and replace/);
       });
   });
-});
-
-describe("HTML replacement", () => {
```

Comment:
Dropping this suite does not affect the percentage of test coverage, and I'll make sure it gets picked up in an E2E test that can genuinely test this.